test/functional/framework.go

```diff
@@ -154,6 +154,24 @@ func (f *FluentdFunctionalFramework) Deploy() (err error) {
 	if err = f.test.Client.Get(f.pod); err != nil {
 		return err
 	}
+	log.V(2).Info("waiting for service endpoints to be ready")
+	serviceUp := false
+	count := 0
+	for count < 10 && !serviceUp {
+		ips, err := oc.Get().WithNamespace(f.test.NS.Name).Resource("endpoints", f.Name).OutputJsonpath("{.subsets[*].addresses[*].ip}").Run()
```

Comment:
can we `oc wait` for a service?

Reply:
no we cant. service doesnt have a conditions in their status
https://github.com/kubernetes/kubernetes/issues/80828